openbas-front/src/admin/components/common/injects/Injects.js

```diff
@@ -487,11 +490,14 @@ const Injects = (props) => {
                         className={classes.bodyItem}
                         style={inlineStyles.inject_type}
                       >
-                        <InjectorContract
-                          variant="list"
-                          config={injectContract?.config}
-                          label={injectorContractName}
-                        />
+                        {injectContract ? (
+                          <InjectorContract
+                            variant="list"
+                            config={injectContract?.config}
+                            label={injectorContractName}
+                          />
+                        ) : <InjectorContract variant="list" label={t('Deleted')} deleted={true} />
+                        }
```

Comment:
Config is not used in the InjectorContract component
You can avoid {true] in react, it's the default value.

```suggestion
                        {injectContract ? (
                          <InjectorContract
                            variant="list"
                            label={injectorContractName}
                          />
                        ) : <InjectorContract variant="list" label={t('Deleted')} deleted />
                        }
```